config/puma.rb

```diff
@@ -44,7 +44,8 @@
 # end
 
 if ENV["VELUM_PORT"].to_i == 443
-  ssl_bind "0.0.0.0", ENV["VELUM_PORT"], key: "/etc/pki/velum.key", cert: "/etc/pki/velum.crt"
+  ssl_bind "0.0.0.0", ENV["VELUM_PORT"], key: "/etc/pki/velum.key", cert: "/etc/pki/velum.crt",
+    ca: "/etc/pki/ca.crt"
```

Comment:
As far as I can tell `ssl_bind` does not support `ca`, it's just ignored: http://www.rubydoc.info/github/puma/puma/Puma%2FConfiguration%2FDSL%3Assl_bind

Reply:
okay. That was part of me trying to get the CA working elsewhere. I can remove this part.